### DISCUSSION TITLE
test sequence without browser reload

### DISCUSSION THREAD

**Discussion by DmitriyTanase** [cypress-io/cypress#16093] [Questions and Help] [answered]
Hi
how can i turn off browser reloading between test suite? I have ng app with routing, every rout of twenty six routs it is solo test suite(it) between thise test suites(it) browser are reload.

**Comment by bahmutov** [comment, marked as answer]
Not sure what you mean exactly, but you can visit the site once and then run all the tests against it

```js
before(() => {
  cy.visit('/')
})
it('works', () => {
   // test commands
})
it('works more', () => {
  // second test commands
})
```

**Reply by DmitriyTanase** [reply]
Thx for answer! Glad to meet with @bahmutov. eah, I found lifecycle hooks, and I thinked - it shall help me. But when I use
```javascript
describe('Test  step 1', () => {
  before('should login', () => {
    cy.visit('/auth/login');
    cy.clickRecaptcha();
  });

  it('should fill step 1 subStep 1', () => {
    // test commands
  });
  it('should fill step 1 subStep 2', () => {
    // test commands
  });
});
``` 
tests does not work, page are freezed. If 'before' replace with 'beforeEach' - all work well, but between 'it' browser do login  job. May bee I incorrect using 'before' hook?

**Comment by DmitriyTanase** [comment]
up

**Comment by DmitriyTanase** [comment]
up
